virtual-desktop/src/app/authentication-manager/login/login.component.ts

```diff
@@ -68,6 +68,8 @@ export class LoginComponent implements OnInit {
     this.errorMessage = '';
     this.expiredPassword = false;
     this.passwordServices = new Set<string>();
+    this.lastActive = new EventEmitter<Number>();
+    window.addEventListener("storage", this.storageEventListener.bind(this));
```

Comment:
added this to coordinate between tabs, listening to storage event, so can detect when last activity update by another tab

Reply:
Additionally, i am thinking using same kind of mechanism
We can generate sessionTimeout event across tabs, right now other tabs just start receiving 401 without user releasing that cookie already invalidated by session expire on other tab.

Reply:
https://github.com/zowe/zlux-app-manager/pull/210/files